demo/api/src/products/entities/product.entity.ts

```diff
@@ -224,4 +226,9 @@ export class Product extends BaseEntity {
     @ManyToMany(() => FileUpload)
     @Field(() => [FileUpload])
     datasheets = new Collection<FileUpload>(this);
+
+    @Property({ type: CustomProductType, nullable: true })
+    @Field(() => CustomProduct, { nullable: true })
+    @CrudField({ inputType: CustomProduct })
```

Comment:
We already have this information in `@Field()`, would it be possible to use this information?

Reply:
yes, but only using ts-morph

0d6a28a